packages/manager/modules/octavia-load-balancer/src/create/template.html

```diff
@@ -90,13 +90,82 @@
             </div>
         </oui-step-form>
 
+        <!-- FLOATING IP -->
+        <oui-step-form
+            data-ng-init="stepFloatingIp = $ctrl.stepper.loadBalancerFloatingIp"
+            data-name="{{:: stepFloatingIp.name}}"
+            data-header="{{:: 'octavia_load_balancer_create_floating_ip_title' | translate }}"
+            data-prevent-next="true"
+        >
+            <div class="mb-4">
+                <p
+                    data-translate="octavia_load_balancer_create_floating_ip_intro"
+                ></p>
+            </div>
+            <oui-spinner data-ng-if="$ctrl.floatingIpLoading"></oui-spinner>
+            <div class="container-fluid" data-ng-if="!$ctrl.floatingIpLoading">
```

Comment:
**nitpick:** Usage of `col-*` should be only within `row` wrapper class in Bootstrap 4.
```suggestion
            <div class="container-fluid row" data-ng-if="!$ctrl.floatingIpLoading">
```